Source/SIMPLib/DataContainers/DataContainerProxy.cpp

```diff
@@ -235,3 +235,64 @@ void DataContainerProxy::setFlags(uint8_t flag, AttributeMatrixProxy::AMTypeFlag
     }
   }
 }
+
+QMap<QString, AttributeMatrixProxy> DataContainerProxy::getattributeMatricies()
+{
+	return attributeMatricies;
```

Comment:
This entire chunk needs to be checked for indentation.
Are these methods supposed to be lowercase until the third word, because I still want to "fix" that.  I'm starting to think I should not.

Reply:
Not sure if we care, but there's no new line at the end of the file.

Reply:
getAttributeMatrices through getAttributeMatrixProxy still need indentation checks.  Also, none of them have the three-line comment blocks before their definitions.